homeworks/hw7/bulls_and_cows/hw7_solution.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+
+def check_guess(users: str, computer: str) -> tuple:
+    bull = 0
+    cow = 0
+    for us in users:
+        for co in computer:
+            if us == co and users.index(us) == computer.index(us):
+                bull += 1
+            elif us == co and users.index(us) != computer.index(us):
+                cow += 1
+    bull_cow = (bull, cow)
+    return bull_cow
```

Comment:
I'm quite sure, that I saw this solution somewhere else.. Did you do it yourself?